README.md

```diff
@@ -78,6 +78,28 @@ And we're back! Use this repo to share and keep track of software, tech, CS, PM,
 | [Two Sigma](https://careers.twosigma.com/careers/JobDetail/New-York-New-York-United-States-Quantitative-Researcher-Internship/10972)                                                                                    | New York, NY                     | Quant Research, [SWE](https://careers.twosigma.com/careers/JobDetail?jobId=11002) |
 | [IMC](https://imc.wd5.myworkdayjobs.com/en-US/invitation/jobs/details/Software-Engineer-Intern----Summer-2023_REQ-01961?q=internship)                                                                                    | Chicago                     | SDE Intern, Quant Research, Trading |
 | [Palantir](https://jobs.lever.co/palantir?commitment=Internship&team=Dev)| Multiple Locations in United States | Early Admissions - Software Engineer, Internship |
+| [84.51°](https://app.ripplematch.com/job/84.51°/2344fff3/) | Cincinnati, Chicago | [Software Engineer - Internship Program 2023](https://app.ripplematch.com/job/84.51°/2344fff3/)|
```

Comment:
Thank you!

3 things:
1) please put new positions at the bottom of the table
2) a number of these postings are duplicates (qualtrics, pinterest, two sigma, etc.)
3) Please link to the actual job posting not the ripple match link

Reply:
I will put a new clean PR, closing this one.